src/native/mqtt_connection.c

```diff
@@ -42,7 +42,7 @@ static struct {
 } s_async_callback = {0};
 
 void s_cache_async_callback(JNIEnv *env) {
-    jclass cls = (*env)->FindClass(env, "software/amazon/awssdk/crt/mqtt/MqttConnection$AsyncCallback");
+    jclass cls = (*env)->FindClass(env, "software/amazon/awssdk/crt/AsyncCallback");
```

Comment:
The async header struct and this function should probably move into crt.c or something, and have a header which externs it, so you can use it in http and wherever else we need callbacks.

Reply:
Fixed